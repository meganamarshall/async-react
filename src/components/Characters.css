.Characters {
    list-style-type: none;
    display: flex;
    flex-wrap: wrap;
  
  li {
    width: 25vw;
    border: 1px solid black;
  }

  img {
    width: 20vw;
  }
}
