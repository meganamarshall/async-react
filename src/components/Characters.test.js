import React from 'react';
import { shallow } from 'enzyme';
import Characters from './Characters';

describe('characters component', () => {
  it('renders the characters', () => {
    const characters = [{
      name: 'rick',
      status: 'alive',
      species: 'human',
      image: 'telkjse.jpg'
    },
    {
      name: 'morty',
      status: 'alive',
      species: 'human',
      image: 'hsdkfl.jpg'
    }];

    const wrapper = shallow(<Characters characters={characters} />);
    expect(wrapper).toMatchSnapshot();
  });
});
