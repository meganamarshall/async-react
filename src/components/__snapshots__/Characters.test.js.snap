// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`characters component renders the characters 1`] = `
<ul
  className="Characters"
>
  <li
    key="0"
  >
    <Character
      character={
        Object {
          "image": "telkjse.jpg",
          "name": "rick",
          "species": "human",
          "status": "alive",
        }
      }
    />
  </li>
  <li
    key="1"
  >
    <Character
      character={
        Object {
          "image": "hsdkfl.jpg",
          "name": "morty",
          "species": "human",
          "status": "alive",
        }
      }
    />
  </li>
</ul>
`;
